src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -300,51 +311,12 @@ public void Initialize(IConsole console)
                 {
                     try
                     {
-                        var result = _runspaceManager.GetRunspace(console, _name);
-                        Runspace = result.Item1;
-                        _nugetHost = result.Item2;
-
                         _initialized = true;
```

Comment:
You're now setting _initialized to true before the host is actually initialized, meaning anything that uses the value may not work correctly.

Reply:
Moved into `GetRunspaceAsync` method so it'll stay same as original.